src/components/AccountDrawer/AuthenticatedHeader.tsx

```diff
@@ -158,6 +160,11 @@ export function PortfolioArrow({ change, ...rest }: { change: number } & IconPro
   )
 }
 
+// LogOut icon is a bit off center
+const LogOutCentered = (props: IconProps) => {
+  return <LogOut {...props} style={{ ...props.style, transform: `translateX(2px)` }} />
```

Comment:
heads up, you will probably be asked to use styled-components for this instead of inline style

Reply:
ah yea i'll change, i'm a fan of inline for simple things wasn't sure how strict we are on that

Reply:
had a similar question on another PR, and hopefully something you can steer us towards consistency on nate 😄